src/qt/bitcoingui.cpp

```diff
@@ -556,8 +556,12 @@ void BitcoinGUI::createActions()
             if (backup_file.isEmpty()) return;
 
             bool wallet_name_ok;
-            //: Title of the Restore Wallet input dialog (where the wallet name is entered)
-            QString wallet_name = QInputDialog::getText(this, tr("Restore Name"), tr("Wallet Name:"), QLineEdit::Normal, "", &wallet_name_ok);
+            /*: Title of pop-up window shown when the user is attempting to
++                restore a wallet. */
```

Comment:
consider include bitcoin-core/gui#663 which fix formatting for gui#629

Reply:
agree